src/TestInfrastructure/TestInfrastructure.csproj

```diff
@@ -28,9 +28,6 @@
   </ItemGroup>
   
   <ItemGroup>
-    <PackageReference Include="FluentAssertions" Version="5.9.0" />
-    <PackageReference Include="Moq" Version="4.18.2" />
-    <PackageReference Include="MSTest.TestFramework" Version="1.4.0" />
```

Comment:
This assembly didn't previously reference a couple of the common packages. However, adding references to them won't do any harm.